contracts/Removal.sol

```diff
@@ -8,14 +8,16 @@ import "@openzeppelin/contracts-upgradeable/token/ERC777/ERC777Upgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC777/IERC777RecipientUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/utils/introspection/ERC1820ImplementerUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/utils/math/SafeMathUpgradeable.sol";
+import "./RestrictedNORI.sol";
 import {RemovalUtils, UnpackedRemovalIdV0} from "./RemovalUtils.sol";
 
 // import "hardhat/console.sol"; // todo
 
 // todo non-transferable/approveable after mint (except by DEFAULT_ADMIN_ROLE)
 // todo disable other mint functions
 
-error TokenIdExists();
+uint256 constant UNIX_EPOCH_2010 = 1_262_304_000;
+uint256 constant UNIX_EPOCH_2050 = 2_524_608_000;
```

Comment:
i'm up for option 1 or 2 (though I think the start time doesn't have to be greater than the block timestamp.... in fact, we probably want the start time to be before the block timestamp because removals may be listed for sale AFTER the start time which is the NRT agreement signing date.  I think most of them would work like this right?)
Maybe @scottynomad wants to give the final opinion... no validation is fine, or maybe we just do the extremely simple check that it's non-zero

Reply:
removed the validation